xml/decoders/Public_Domain_dccdoma_UNI16ARD_NAV.xml

```diff
@@ -550,8 +550,8 @@
                     <choice xml:lang="cs">P</choice>
                 </enumChoice>
                 <enumChoice choice="Not addressed" value="16" >
-                    <choice>×</choice>
-                    <choice xml:lang="cs">×</choice>
+                    <choice>×  </choice>
```

Comment:
Why adding an extra space here?
I'm not against it, I'm asking since I don't understand.

Reply:
It's a dirty trick.
LaF Metal has an ugly bold font. GTK does not change the background colour in the fill-ins.
Nimbus is fine. Unfortunately, he has a problem with the one-letter combo-box. 
The combo box is too short to display the one-letter value. Only three dots are displayed. Value `<choice>×  </choice>` make it wide enough.
Do you have a better idea?
![Nimbus](https://user-images.githubusercontent.com/20986698/133488881-638e8a83-8128-47a3-a8ab-8722c3e353b1.png)

Reply:
> Do you have a better idea?

I have asked about it on the JMRI developers list:
https://jmri-developers.groups.io/g/jmri/message/6056